jenkins/daily-tags

```diff
@@ -32,6 +32,8 @@ try {
               rm -rf "$PYTHONUSERBASE"
               yum install -y python3-pip python3-devel python3-setuptools
               mkdir -p /local/tmp
+              python -m venv /local/tmp/venv
```

Comment:
Since this is only for ubuntu, I would protect it with a `case $ARCHITECTURE in ubuntu24*)...`.